CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added a DOI format and organization check to detect [American Physical Society](https://journals.aps.org/) journals to copy the article ID
 to the page field for cases where the page numbers are missing. [#7019](https://github.com/JabRef/jabref/issues/7019)
 - We added a new fetcher to enable users to search jstor.org [#6627](https://github.com/JabRef/jabref/issues/6627)
+- We added an error message in the EntryTypeViewModel that gets displayed to the user if a fetched entry is empty. [#7000](https://github.com/JabRef/jabref/issues/7000)
```

Comment:
```suggestion
- We added an error message in the New Entry dialog that is shown in case the fetcher did not find anything . [#7000](https://github.com/JabRef/jabref/issues/7000)
```
The changelog is more intended towards the normal end users of JabRef, so it should not be too technical.